docker-compose.hosted.geth.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.5'
+# Use this script for running up nightfall_3 in 'ropsten' mode with local
+# Use it as an override file for docker-compose.yml
+# See the readme for more information.
+services:
+  client1:
+    environment:
+      BLOCKCHAIN_URL: $HOSTED_ROPSTEN_NODE
```

Comment:
I would recommend changing `HOSTED_ROPSTEN_NODE` to `ROPSTEN_NODE`.

Reply:
done

Reply:
There are still artifacts of this. Could you correct everything to do with 'hosted` in the naming please
- files with hosted-geth in their names
- npm script `hosted-geth-test`
- `--hosted-geth` flag

Reply:
done